pkg/client/src/app/pages/controls/tags/tags.tsx

```diff
@@ -140,6 +145,16 @@ export const Tags: React.FC = () => {
           .flat()
           .filter((tag) => tag && tag.name)
           .map((tag) => ({ key: tag?.name, value: tag?.name }))
+          .concat(
+            tagTypes?.map((tagType) => ({
+              key: tagType?.name,
+              value: tagType?.name,
+            }))
+          )
+          .sort((a, b) => {
+            if (a.value && b.value) return a?.value.localeCompare(b?.value);
+            else return 0;
```

Comment:
Huh, I didn't realize single-line if could be used with else like this.

Reply:
I think you are right to point this out as odd syntax. I moved to curly brackets for else statements. Looks to be more maintainable.